envoy/network/address.h

```diff
@@ -100,6 +100,38 @@ class Ip {
    */
   virtual bool isUnicastAddress() const PURE;
 
+  /**
```

Comment:
Can you add the IP prefixes into these doc comments for each type? It makes it easier to read without having to refer to the RFC to remember which one is which.